timescale-container/test/sql/count-imported-records.sql

```diff
@@ -0,0 +1 @@
+SELECT count(*) FROM mtlusovini.etdevice;
```

Comment:
i am not sure i would refer to lusovini in a public repo.

Reply:
got rid of it, thx!